langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaChatModel.java

```diff
@@ -45,10 +45,14 @@ public OllamaChatModel(String baseUrl,
                            List<String> stop,
                            String format,
                            Duration timeout,
-                           Integer maxRetries) {
+                           Integer maxRetries,
+                           Boolean logRequests,
+                           Boolean logResponses) {
         this.client = OllamaClient.builder()
                 .baseUrl(baseUrl)
                 .timeout(getOrDefault(timeout, ofSeconds(60)))
+                .logRequests(logRequests)
```

Comment:
```suggestion
                .logRequests(getOrDefault(logRequests, false))
```
Please set defaults in all models, for requests and responses